pkg/types/stack.go

```diff
@@ -57,6 +59,7 @@ type StackList struct {
 // StackSpec defines the desired state of Stack
 type StackSpec struct {
 	Metadata
+	Templates      []string                         `json:"templates"`
```

Comment:
A couple variations to consider:
* Refine the Stack object itself and put the Templates and PropertyValues in that object, instead of the Spec.  Then for the serialization into raft, we could omit the Spec and just store the Templates/Properties if we want to avoid duplicate data.
* Nest the Services/Secrets/etc one layer deeper into the Spec as "rendered" fields to make the delineation clearer between mutable and immutable fields